src/Sarif/NullableEnum.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// A wrapper class for Nullable Enum
+    /// </summary>
+    public static class NullableEnum
```

Comment:
>NullableEnum [](http://example.com/codeflow?start=24&length=12)

This is a helper class I added for simplify code related to Nullable Enum, we can add more methods if helpful

Reply:
this is no longer needed, deleted.